From 63a9895e4b0c5c6b94e409b8019ff85d615ae24e Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 16 Aug 2017 15:52:06 +0800
Subject: [PATCH 2369/5242] MLK-16201 arm64: dts: freescale: imx8mq: add dcss
 and hdmi power domain

commit  c56a287f3142414022f4d4d499af26999f048077 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX8MQ, DCSS and HDMI are in display mix power domain,
assgin them to display mix power domain.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 0c9f79f..e426f40 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -346,6 +346,7 @@
 		disp-mode = <16>;			/* <default mode: #16>
 							 * #16: 1920x1080p@60Hz 16:9
 							 */
+		power-domains = <&power 7>;
 		status = "disabled";
 	};
 
@@ -923,6 +924,7 @@
 					 * #95: 3840x2160p@30Hz 16:9
 					 * #97: 3840x2160p@60Hz 16:9
 					 */
+		power-domains = <&power 7>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

